ember_debug/utils/ember.js

```diff
@@ -3,7 +3,13 @@ import { emberSafeRequire } from 'ember-debug/utils/ember/loader';
 let Ember;
 
 try {
-  Ember = requireModule('ember').default;
+  Ember = requireModule('ember/barrel').default;
+} catch {
+  Ember = window.Ember;
```

Comment:
Shouldn't that catch be empty so `requireModule('ember').default` prevails over `window.Ember`?